src/Engine/ProtoCore/Lang/BuiltInFunctionEndPoint.cs

```diff
@@ -1731,7 +1734,7 @@ internal static StackValue Remove(StackValue sv1, StackValue sv2, ProtoCore.DSAS
 
             var svArray = runtimeCore.Heap.ToHeapObject<DSArray>(sv1).Values.ToArray();
             int length = svArray.Length;
-            int indexToBeRemoved = (int)sv2.opdata;
+            int indexToBeRemoved = (int)sv2.IntegerValue;
```

Comment:
Another casting that may not be needed if we made `IntegerValue` an `int` instead of `Int64`.